src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -83,9 +83,10 @@ internal string GetDebuggerDisplay()
                 usingDirectives = inUsing ? default(SyntaxList<UsingDirectiveSyntax>) : compilationUnit.Usings;
                 externAliasDirectives = compilationUnit.Externs;
             }
-            else if (declarationSyntax.Kind() == SyntaxKind.NamespaceDeclaration)
+            else if (declarationSyntax.Kind() == SyntaxKind.NamespaceDeclaration ||
```

Comment:
>declarationSyntax.Kind() == SyntaxKind.NamespaceDeclaration || [](start = 21, length = 62)

nit: `declarationSyntax.Kind() is (SyntaxKind.NamespaceDeclaration or SyntaxKind.SingleLineNamespaceDeclaration)`
also applies in other places in this PR

Reply:
good call :)